tests/integration/test_spans.py

```diff
@@ -726,131 +727,139 @@ def test_span_ingestion(
 
     now_timestamp = int(now.timestamp())
     expected_timestamp = int(end.timestamp())
-    expected_span_metrics = [
-        {
-            "name": "c:spans/usage@none",
-            "org_id": 1,
-            "project_id": 42,
-            "retention_days": 90,
-            "tags": {},
-            "timestamp": expected_timestamp,
-            "type": "c",
-            "value": 3.0,
-            "received_at": time_after(now_timestamp),
-        },
-        {
-            "name": "c:spans/usage@none",
-            "org_id": 1,
-            "project_id": 42,
-            "retention_days": 90,
-            "tags": {},
-            "timestamp": expected_timestamp + 1,
-            "type": "c",
-            "value": 3.0,
-            "received_at": time_after(now_timestamp),
-        },
-        {
-            "name": "d:spans/duration@millisecond",
-            "org_id": 1,
-            "project_id": 42,
-            "retention_days": 90,
-            "tags": {
-                "file_extension": "js",
-                "span.category": "resource",
-                "span.description": "https://example.com/*/blah.js",
-                "span.domain": "example.com",
-                "span.group": "8a97a9e43588e2bd",
-                "span.op": "resource.script",
+    expected_span_metrics = (
```

Comment:
This is the test that was flaky because sometimes, there were more than two `usage` entries (split on bucket boundary).